input/jgr_grid.cfg

```diff
@@ -151,7 +145,7 @@ Phi_global                  = 1.0       # Mantle melt fraction initial guess
 CH_ratio                    = 1.0       # C/H ratio
 hydrogen_earth_oceans       = 6.0       # Hydrogen inventory in units of equivalent Earth oceans
 nitrogen_ppmw               = 2.0       # Nitrogen inventory in ppmw relative to mantle mass
-sulfur_ppmw                 = 200.0     # Nitrogen inventory in ppmw relative to mass of melt
+sulfur_ppmw                 = 200.0     # Sulfur inventory in ppmw relative to mass of melt
```

Comment:
This should be 0, or do you want to repeat the grid for the revision, including S?